sdk/core/System.ClientModel/tests/nullableenabledclient/System.ClientModel.Tests.NullableEnabledClient.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This will be removed once https://github.com/Azure/azure-sdk-for-net/issues/41554 is addressed.